osquery/filesystem/windows/fileops.cpp

```diff
@@ -625,11 +625,7 @@ bool PlatformFile::isSpecialFile() const {
   return (GetFileType(handle_) != FILE_TYPE_DISK);
 }
 
-static Status isUserCurrentUser(PSID user) {
-  if (!IsValidSid(user)) {
-    return Status(-1, "Invalid SID");
-  }
-
+Status getCurrentUserInfo(PTOKEN_USER& info) {
```

Comment:
The getCurrentUserInfo() and isUserCurrentUser() are only used by tables.  I wish there were separate libraries for table support routines like this, as it makes the core harder and harder to separate and whittle down.  It appears that both of these routines are not used outside the certificates table, so perhaps it can be contained in there for now?

Reply:
I somewhat disagree. 

I think there's value in having some core table helper functions about. These sound like great candidates to me.  (Certainly we have equivalents in our go tables)

Reply:
@packetzero I see what you're saying- In this particular case, I feel like although the certificates table is the only caller, these functions are general enough that it makes sense to keep them in a general library of routines available to windows tables, including future ones, so I tend to agree with @directionless here.

Reply:
yeah, not a huge deal.  I'm all for common routines, we need to start thinking about the monolith that is osquery and how to separate core from tables at some point.  When builds have stabilized, we can think about moving table-specific helpers into libs separate from core.